frontend/src/components/Form/PaginatedSelectMenu.tsx

```diff
@@ -39,6 +39,8 @@ export interface PaginatedSelectMenuProps<
   control?: Control<T, unknown>;
   /** If set to true, the user has to select an option before the form can be completed (default false).*/
   required?: boolean;
+  /** Force a selected option. */
```

Comment:
Please elaborate a bit, does this mean you cannot select at all?

Reply:
Essentially yes. It seems like you can't set a option that is selected by default without tracking the selected component externally to the react-select component.

Reply:
Please add this to documentation.